core/templates/dev/head/exploration_editor/feedback_tab/ThreadDataService.js

```diff
@@ -69,6 +70,11 @@ oppia.factory('threadDataService', [
         }
       }
     });
+    var payload = {
+      exploration_id: _expId,
+      thread_id: threadId
+    };
+    $http.post(_FEEDBACK_THREAD_VIEW_EVENT_URL, payload);
```

Comment:
Ok, I just checked. Actually function which called when thread is opened is `setActiveThread() in FeedbackTab.js`. This function makes call to `fetchMessages() function in ThreadDataServices.js` which then calls `_fetchMessages()` and currently I have put post request in implementation of this `_fetchMessages`. So where to actually put it, according to you ?
In `setActiveThread()` or in `fetchMessages()` ?